features/fixtures/policy_and_engagement_signup_content_item.json

```diff
@@ -0,0 +1,43 @@
+{
+  "base_path": "/policy-papers-and-consultations/email-signup",
+  "content_id": "5a4dc517-57cf-4dd6-873f-f1d29f6d540c",
+  "document_type": "finder_email_signup",
+  "title": "Policy papers and consultations",
+  "description": "You'll get an email each time policy papers and consultations are published.",
+  "details": {
+    "filter": {
+      "content_purpose_supergroup": "policy-papers-and-consultations"
+    },
+    "email_filter_by": null,
+    "email_filter_name": null,
+    "subscription_list_title_prefix": "Policy papers and consultations",
+    "email_filter_facets": [
+      {
+        "facet_id": "people",
+        "facet_name": "people"
+      },
+      {
+        "facet_id": "organisations",
+        "facet_name": "organisations"
+      },
+      {
+        "facet_id": "document_type",
```

Comment:
This'll need to be `content_store_document_type`, and it should support option_lookup behaviour like the finder (otherwise we'll pass `content_store_document_type=policy_papers` to email-alert-api when we should be passing `content_store_document_type=[impact_assessment,case_study,policy_paper]`).